app/controllers/host_controller.rb

```diff
@@ -179,7 +179,7 @@ def update
         valid_host = find_record_with_rbac(Host, params[:id])
         set_record_vars(valid_host, :validate) # Set the record variables, but don't save
         if valid_record? && set_record_vars(@host) && @host.save
-          AuditEvent.success(build_saved_audit_hash_angular(old_host_attributes, @host, false))
+          AuditEvent.success(build_saved_audit(@host, :new => @host.attributes.clone, :current => old_host_attributes))
```

Comment:
In a followup PR, I think I'm going to change all `build_saved_audit` to `audit_saved` or something, and move the `AuditEvent.success` inside it, since every caller to `build_saved_audit` also calls `AuditEvent.success`